src/Nano/Address.h

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2019 Mart Roosmaa.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PublicKey.h"
+
+#include <string>
+
+namespace TW::Nano {
+
+enum AddressPrefix {
+    nano,
+    xrb,
+};
+
+class Address {
```

Comment:
This question is possibly directed at the UI team somewhat. How do you prefer for the Nano::Address class to behave in regards to prefixes? Basically Nano addresses are made up of `<prefix>_<checksum><pubkey>`, the old, still widely used, prefix is `xrb_...`; the new which wallets are slowly transitioning to is `nano_...`. Technically the prefix mean almost nothing, but users still get somewhat confused when the same address is displayed at different places with different prefixes.

This version of the code treats addresses with different prefixes as different objects (when the equality operator is concerned). Which, after coming back to after the weekend, I'm starting to think might not be such a good idea.

Currently I'm thinking maybe the best option would be to accept both prefixes in the Nano::Address constructor, but not store the prefix used in the instance data and just always output the newest prefix, eg a `xrb_...` address would be automatically converted to `nano_...`. For this case, in the UI, there should be some logic to detect if the Nano::Address::string() output is equal to the original address passed in, then the UI should inform the user that the old address was automatically converted to the new address.

Reply:
@roosmaa I believe the Nano addresses are `<prefix>_<pubkey><checksum>`.

Reply:
Yes, the checksum is at the end. Tired today 😫

Reply:
Does other nano wallets support sending coins to `nano_` address? we always prefer new address

Reply:
I think most do, but some don't. @zhyatt can probably answer this more accurately.

Reply:
@hewigovens @roosmaa With the next version of the node (V19) coming soon, it will be emitting `nano_` addresses for all RPC calls but continue to support incoming requests with both `nano_` and `xrb_` addresses. As a result, the various Nano wallets are making updates to support `nano_` if they don't already. The wallet should allow entering/pasting/scanning QR codes for addresses with either prefix, but only display/emit `nano_` addresses. Does that provide enough context?

Reply:
Ok, in that case I'll strip the prefix storage, and always emit `nano_`. Parsing `xrb_` will still work, but the parsed address will be converted to `nano_` automatically.